# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/PWebRenderBridge.ipdl
# Commit: 5506d1803e15
# Full Hash: 5506d1803e158df5504bc74a7818cb78dca6420c
# Author: Kris Taeleman <ktaeleman@mozilla.com>
# Date: 2019-10-16 05:20:00
# Description:
#   Bug 1585801 - Fixing crash when folder is not writable and refactored logic. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D48999
# ==============================================================================

diff -r f24ddb55b48b -r 5506d1803e15 gfx/layers/ipc/PWebRenderBridge.ipdl
--- a/gfx/layers/ipc/PWebRenderBridge.ipdl	Sat Oct 12 23:50:45 2019 +0000
+++ b/gfx/layers/ipc/PWebRenderBridge.ipdl	Tue Oct 15 17:03:22 2019 +0000
@@ -73,6 +73,9 @@
   // Save the frame capture to disk
   async Capture();
 
+  // Enable/Disable transaction logging
+  async SetTransactionLogging(bool aValue);
+
   // Replacement for PCompositorBridge::SyncWithCompositor, but for WR. We need
   // it on PWebRenderBridge because it associated with a particular top-level
   // window, and PCompositorBridge doesn't allow doing that in a secure manner.